jib-gradle-plugin/src/test/java/com/google/cloud/tools/jib/gradle/GradleProjectPropertiesTest.java

```diff
@@ -374,6 +379,7 @@ public void testCreateContainerBuilder_noClassesFiles() throws InvalidImageRefer
         .thenReturn(new TestFileCollection(ImmutableSet.of(nonexistentFile)));
     gradleProjectProperties.createJibContainerBuilder(
         JavaContainerBuilder.from(RegistryImage.named("base")), ContainerizingMode.EXPLODED);
+    gradleProjectProperties.waitForLoggingThread();
```

Comment:
I noticed we never call this in `MavenProjectPropertiesTest`, should we be?

Reply:
`MavenProjectPropertiesTest` doesn't have any `verify(mockLog)`, so need there (yet).